lib/round.rb

```diff
@@ -0,0 +1,104 @@
+class Round
+
+  attr_reader :deck , :turns , :number_correct
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+    @turn_count = 0
+
+
+  end
+
+  def current_card
+    deck.cards[0]
+  end
+
+  def take_turn(guess)
+
+    turn = Turn.new(guess, current_card)
+    turns << turn
+    if turn.correct?
+      @number_correct += 1
+    end
+    deck.cards.rotate!
+    return turn
+    # turn
```

Comment:
Looks like you went with an explicit return here instead of implicit - it can be weird not using the `return` keyword but it is totally fine to do so in Ruby! Try to get used to the joys of "letting go of the handlebars" when you can ;)